.test-env

```diff
@@ -1,6 +1,6 @@
 # Configs for testing repo download:
 SDK_TESTING_URL="https://github.com/algorand/algorand-sdk-testing"
-SDK_TESTING_BRANCH="V2"
+SDK_TESTING_BRANCH="update-indexer-mocks"
```

Comment:
```suggestion
SDK_TESTING_BRANCH="V2"
```
TODO: revert when https://github.com/algorand/algorand-sdk-testing/pull/308 is merged